src/features/Dashboard/Dashboard.tsx

```diff
@@ -57,11 +51,11 @@ export default function Dashboard() {
           <div className="Dashboard-mobile-walletBreakdown">
             <WalletBreakdown />
           </div>
-          {featureFlags.MYC_MEMBERSHIP && (
+          {/* {featureFlags.MYC_MEMBERSHIP && (
             <div className="Dashboard-mobile-section Dashboard-mobile-tokenList">
               <MembershipPanel />
             </div>
-          )}
+          )} */}
```

Comment:
Can we deactivate the flag or delete the code instead ?
Leaving commented code opens a big can of worms.

Reply:
For this release, im going to leave membership in. After this release, nav should be merged before the next release so ill remove it for that release.